tools/build/download-kube-bin.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build Kubernetes release images. This will build the server target binaries,
+# and create wrap them in Docker images, see `make release` for full releases
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
+source "${KUBE_ROOT}/tools/build/lib/common.sh"
+
+KUBE_VERSION=v1.21.0
+
+KUBERNETES_RELEASE_URL="${KUBERNETES_RELEASE_URL:-https://dl.k8s.io}"
+DOWNLOAD_URL_PREFIX="${KUBERNETES_RELEASE_URL}/${KUBE_VERSION}"
+
+KUBE_DOWNLOAD_DIR="${KUBE_DOWNLOAD_DIR:-"${OUT_DIR}/kube"}"
+
+function download_tarball() {
```

Comment:
Are the downloaded tarballs' hash not checked against some known value? I'm not 100% sure of the value of doing so - I'm only asking since [this is currently done](https://github.com/kubernetes/cloud-provider-gcp/blob/master/WORKSPACE#L103-L110) with Bazel.

Reply:
implemented. Thanks for the hint.